pkg/decode/decode_protobuf.go

```diff
@@ -141,6 +141,10 @@ func RecordToMap(fr *model.Record) config.GenericMap {
 			out["XlatSrcAddr"] = model.IP(fr.Metrics.AdditionalMetrics.TranslatedFlow.Saddr).String()
 			out["XlatDstAddr"] = model.IP(fr.Metrics.AdditionalMetrics.TranslatedFlow.Daddr).String()
 		}
+		if fr.Metrics.AdditionalMetrics.FlowEncrypted || fr.Metrics.AdditionalMetrics.FlowEncryptedRet != 0 {
+			out["EncryptedFlow"] = fr.Metrics.AdditionalMetrics.FlowEncrypted
+			out["EncryptedFlowRet"] = fr.Metrics.AdditionalMetrics.FlowEncryptedRet
```

Comment:
I'm not a huge fan of the field naming here. Since the feature is called `IPSec` why not using `IPSecEncrypted` (or just `IPSec`) and `IPSecCode` ?